web_src/js/utils/dom.ts

```diff
@@ -314,6 +314,7 @@ export function replaceTextareaSelection(textarea: HTMLTextAreaElement, text: st
 export function createElementFromHTML<T extends HTMLElement>(htmlString: string): T {
   htmlString = htmlString.trim();
   // some tags like "tr" are special, it must use a correct parent container to create
+  // eslint-disable-next-line github/unescaped-html-literal -- FIXME: maybe we need to use other approaches to create elements from HTML, e.g. using DOMParser
   if (htmlString.startsWith('<tr')) {
```

Comment:
Why is this condition even in this function? It seems like it has no place here and should be outside.

Reply:
Sorry I didn't get your point. Could you elaborate?

Reply:
I mean this `tr are special` thing, does it really have to be in here?

Reply:
> I mean this `tr are special` thing, does it really have to be in here?

It is, that's why it is here, you can blame the history.

And that's why I added the FIXME. IMO it is not in this PR's scope.